packages/babel-plugin-transform-runtime/scripts/runtime-corejs2-definitions.js

```diff
@@ -0,0 +1,146 @@
+// This file isn't maintaned anymore.
+// Improvements should go in the babel-plugin-polyfill-corejs2 package.
+
+export default {
```

Comment:
These two files have just been moved from `src` to `scripts`. GitHub shows them as new because I changed the indentation.